tensorflow_datasets/image_classification/cbis_ddsm.py

```diff
@@ -709,8 +709,8 @@ def _sample_positive_patches(image,
     # Determine the region where random samples should be sampled from.
     max_h, min_h = max(abnorm_h, patch_size[0]), min(abnorm_h, patch_size[0])
     max_w, min_w = max(abnorm_w, patch_size[1]), min(abnorm_w, patch_size[1])
-    min_y = abnorm_y - int((1.0 - min_overlap_threshold) * max_h)
-    min_x = abnorm_x - int((1.0 - min_overlap_threshold) * max_w)
+    min_y = abnorm_y - int(max_h - min_h) - int((1.0 - min_overlap_threshold) * min_h)
```

Comment:
Could this be simplified into `- max_h + min_overlap_threshold * min_h` ?

Reply:
OK so the simplification is of course correct, sorry about that.

Regarding the comment from the internal discussion, I will try to clarify:
In the initial implementation, min_overlap_threshold = 0.9 and abnorm_x and abnorm_y are the coordinates of the upper left corner of the abnormality bounding box. So the **minimum window position** is min_x = abnorm_x - 22.4. This is too close to the abnormality x: All possible random windows will start at a maximum 22 pixel distance from the ibnormality.
Instead, the actual minimum window x and y should indeed come closer the the upper-left corner of the mammogram which will place the abnormality further down-right in the patch..

Reply:
@jpuigcerver FYI